sdk/libs-haskell/da-hs-base/src/Options/Applicative/Extended.hs

```diff
@@ -27,6 +28,12 @@ data YesNoAuto
     | Yes
     deriving (Eq, Ord, Show)
 
+-- | Convert YesNoAuto to Maybe Bool
+yesNoAutoToMaybe :: YesNoAuto -> Maybe Bool
+yesNoAutoToMaybe No = Just False
+yesNoAutoToMaybe Yes = Just True
+yesNoAutoToMaybe Auto = Nothing
```

Comment:
TODO: Drop this, refactor flags using this to specify a default and properly use yesNoAutoFlag